CHANGELOG.md

```diff
@@ -8,10 +8,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 - `DofHandler` now supports fields on subdomains and mixed grids. ([#667][github-667])
+- Support for embedded elements in `CellValues`. ([#651][github-651])
+- Support for vectorized interpolations with dimension different from the spatial 
+  dimension in `CellValues`. ([#651][github-651])
+
 ### Removed
 - **BREAKING**: `MixedDofHandler` has been renamed to `DofHandler`. Update by replacing 
 `MixedDofHandler` to `DofHandler`. ([#667][github-667])
-
+- **BREAKING**: All `CellValues` have a finer granularity on the different types of
+  dimensions used. Custom `CellValues` must be updated to respect the new type 
+  parameters. ([#651][github-651])
```

Comment:
Should it also mention that the parameterization is changed, and dispatches must be updated if dispatching on e.g. `T` or `RefShape`?

Reply:
Let's leave the discussion until https://github.com/Ferrite-FEM/Ferrite.jl/pull/627 is merged (+ I merged new master).